state/addmachine.go

```diff
@@ -824,7 +824,7 @@ func (st *State) maintainControllersOps(mdocs []*machineDoc, currentInfo *Contro
 // according to the specified placement directives until the placement list is
 // exhausted; thereafter any new machines are started according to the constraints and series.
 func (st *State) EnableHA(
-	numControllers int, cons constraints.Value, series string, placement []string,
+	numControllers int, cons constraints.Value, series string, placement []string, machineId string,
```

Comment:
Could you add a description of what machineId does to the docstring?

Reply:
done